res/controllers/Pioneer-DDJ-400.midi.xml

```diff
@@ -3017,6 +2853,40 @@
                 <minimum>0.5</minimum>
             </output>
 
+            <!-- Beatjump (light pads 7&8 when shift is pressed) -->
```

Comment:
"light pads 7&8 when shift is pressed"

What is the reason for this?

Reply:
Replicates rekord box's light behavior for [changing the beatjump sizes](https://github.com/jusko/manual/blob/controller_pioneer_ddj_400/source/hardware/controllers/pioneer_ddj_400.rst#beat-jump-mode).